kgitb/routes.py

```diff
@@ -7,58 +7,58 @@
 
 
 def check_pr(commits_url, comments_url, status_url):
-    github.update_status(status_url, "pending",
-                         "The KGB is reviewing your commits.")
+    github.update_status(status_url, 'pending',
+                         'The KGB is reviewing your commits.')
     all_errors = 0
     messages = []
     for commit in github.get_commits(commits_url):
-        message = commit["commit"]["message"]
+        message = commit['commit']['message']
         errors = rules.apply_rules(message)
         if len(errors) > 0:
-            sha = commit["sha"]
-            messages.append("\n".join([sha, github.quote(message), ""] +
+            sha = commit['sha']
+            messages.append('\n'.join([sha, github.quote(message), ''] +
                                       errors))
             all_errors += len(errors)
 
     log.debug(all_errors)
     github.upsert_comment(comments_url, messages)
     if all_errors == 0:
-        github.update_status(status_url, "success", "We are proud of you!")
+        github.update_status(status_url, 'success', 'We are proud of you!')
     elif all_errors == 1:
-        github.update_status(status_url, "error", "Fix the error!")
+        github.update_status(status_url, 'error', 'Fix the error!')
     else:
-        github.update_status(status_url, "error",
-                             "Fix the {} errors!".format(all_errors))
+        github.update_status(status_url, 'error',
+                             'Fix the {} errors!'.format(all_errors))
 
 
 def github_pr():
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a docstring to this function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)